hathor/conf/testnet.py

```diff
@@ -55,36 +53,6 @@
         cp(1_600_000, bytes.fromhex('00000000060adfdfd7d488d4d510b5779cf35a3c50df7bcff941fbb6957be4d2')),
     ],
     FEATURE_ACTIVATION=FeatureActivationSettings(
-        evaluation_interval=40_320,
-        default_threshold=30240,
-        features={
-            Feature.NOP_FEATURE_4: Criteria(
-              bit=0,
-              start_height=3_386_880,  # N (right now the best block is 3_346_600 on testnet)
-              timeout_height=3_467_520,  # N + 2 * 40320 (4 weeks after the start)
-              minimum_activation_height=3_507_840,  # N + 3 * 40320 (6 weeks after the start)
-              lock_in_on_timeout=False,
-              version='0.57.0',
-              signal_support_by_default=True
-            ),
-            Feature.NOP_FEATURE_5: Criteria(
-              bit=1,
-              start_height=3_386_880,  # N (right now the best block is 3_346_600 on testnet)
-              timeout_height=3_467_520,  # N + 2 * 40320 (4 weeks after the start)
-              minimum_activation_height=0,
-              lock_in_on_timeout=True,
-              version='0.57.0',
-              signal_support_by_default=False
-            ),
-            Feature.NOP_FEATURE_6: Criteria(
-              bit=2,
-              start_height=3_386_880,  # N (right now the best block is 3_346_600 on testnet)
-              timeout_height=3_467_520,  # N + 2 * 40320 (4 weeks after the start)
-              minimum_activation_height=0,
-              lock_in_on_timeout=False,
-              version='0.57.0',
-              signal_support_by_default=False
-            )
-        }
+        default_threshold=15_120,  # 15120 = 75% of evaluation_interval (20160)
```

Comment:
I guess it's missing the `evaluation_internal=20_160`, isn't it?

Reply:
Nope, `20160` is the default value in the configuration class, so I'm just removing an override. For mainnet it's the same, this key is not being set. Now both mainnet and testnet don't need it.

Reply:
Got it. Thanks!